src/decode-ipv6.c

```diff
@@ -453,6 +453,12 @@ DecodeIPV6ExtHdrs(ThreadVars *tv, DecodeThreadVars *dtv, Packet *p,
                     plen -= hdrextlen;
                     break;
                 }
+                if (p->ip6eh.fh_more_frags_set != 0 && plen & 7) {
```

Comment:
what is the `& 7` here?

Reply:
That is the check that each fragment has a length being an integer multiple of 8 octets long.

Reply:
I can write `plen % 8 != 0` if that is more readable and the compiler should get it optimized, right ?

Reply:
Yeah that helps my tiny brain to also understand it ;)

Reply:
Done